resources/views/components/delete-modal.blade.php

```diff
@@ -0,0 +1,16 @@
+<div class="modal fade" id="deletConfirmation" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="deletConfirmationLabel" aria-hidden="true">
+    <div class="modal-dialog">
+    <div class="modal-content">
+        <div class="modal-header">
+        <h1 class="modal-title fs-5" id="deletConfirmationLabel">Are You Sure to Delete the {{ $module  }} ?? </h1>
```

Comment:
explain how you dynamically changing the title. I assume you are not using an additional routes and view for re render this component

Reply:
im passing the name to the component when rendering the component like this

![image](https://github.com/user-attachments/assets/0ec21357-c2bb-4036-9688-d63399b90ad1)

Reply:
can you show me the output as screen shot

Reply:
Sure

employee module
![image](https://github.com/user-attachments/assets/e56d6c6c-4abe-4e32-889a-bb0784c8f700)

Company Module
![image](https://github.com/user-attachments/assets/5b52050e-da67-4ca2-8b69-bd4f20635578)

Reply:
there are multiple rows, which recode are you trying to delete?, how will you mention that?

for example refer below
![image](https://github.com/user-attachments/assets/38d51a6d-4eee-4453-afbc-0ed44d5801ec)

Reply:
we can set the message and name dynamically...using jQuery after the component is rendered seems like i havent implemented it...shall implement it?

Reply:
don't on this PR because of too many changes. we shall make it later.

Reply:
Okay sure